### PR TITLE
Display KHR loader as Khronos

### PR DESCRIPTION
**Pull request by BastiaanOlij**
Display the description of the KHR loader as Khronos.

We should find a way to add a description in the popup of the tickboxes in the export preset dialog so we can provide a clearer description that the Khronos loader is the standard loader.

### CODE REVIEW

File: demo/addons/godotopenxr/.export/godot_openxr_editor_plugin.gd [hunk 1]
```diff
@@ -15,7 +15,7 @@ func _enter_tree():
 	meta_export_plugin = preload("meta/godot_openxr_meta_editor_export_plugin.gd").new("meta", plugin_version)
 	pico_export_plugin = preload("pico/godot_openxr_pico_editor_export_plugin.gd").new("pico", plugin_version)
 	lynx_export_plugin = preload("lynx/godot_openxr_lynx_editor_export_plugin.gd").new("lynx", plugin_version)
-	khr_export_plugin = preload("khr/godot_openxr_khr_editor_export_plugin.gd").new("khr", plugin_version)
+	khr_export_plugin = preload("khr/godot_openxr_khr_editor_export_plugin.gd").new("khronos", plugin_version)
```
Comment (m4gr3d) [inline demo/addons/godotopenxr/.export/godot_openxr_editor_plugin.gd:18]: You would also need to update the folder names and the maven central dependency from 'khr' to 'khronos' to match, as the `vendor` name argument is used to determine the location of the aar binaries and of the maven central dependencies.

**Review by m4gr3d — CHANGES_REQUESTED** [review]

File: demo/addons/godotopenxr/.export/godot_openxr_editor_plugin.gd (hunk 1 above)
Comment (BastiaanOlij) [reply demo/addons/godotopenxr/.export/godot_openxr_editor_plugin.gd]: I think I got it all now.

File: godotopenxrlynx/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -19,7 +19,7 @@
     <application>
         <meta-data
             android:name="org.godotengine.plugin.v2.GodotOpenXRLynx"
-            android:value="org.godotengine.openxrloaders.khr.GodotOpenXRLynx" />
+            android:value="org.godotengine.openxrloaders.lynx.GodotOpenXRLynx" />
```
Comment (BastiaanOlij) [inline godotopenxrlynx/src/main/AndroidManifest.xml:22]: also found typo in lynx manifest :)

File: demo/addons/godotopenxr/.export/godot_openxr_editor_plugin.gd [hunk 3]
```diff
@@ -7,32 +7,32 @@ var globals = preload("globals.gd")
...
 	
 	# Initializing the export plugins
-	meta_export_plugin = preload("meta/godot_openxr_meta_editor_export_plugin.gd").new(globals.META_VENDOR_NAME, plugin_version)
-	pico_export_plugin = preload("pico/godot_openxr_pico_editor_export_plugin.gd").new(globals.PICO_VENDOR_NAME, plugin_version)
-	lynx_export_plugin = preload("lynx/godot_openxr_lynx_editor_export_plugin.gd").new(globals.LYNX_VENDOR_NAME, plugin_version)
-	khr_export_plugin = preload("khr/godot_openxr_khr_editor_export_plugin.gd").new(globals.KHRONOS_VENDOR_NAME, plugin_version)
+	meta_export_plugin = preload("meta/godot_openxr_meta_editor_export_plugin.gd").new("meta", plugin_version)
+	pico_export_plugin = preload("pico/godot_openxr_pico_editor_export_plugin.gd").new("pico", plugin_version)
+	lynx_export_plugin = preload("lynx/godot_openxr_lynx_editor_export_plugin.gd").new("lynx", plugin_version)
+	khronos_export_plugin = preload("khronos/godot_openxr_khronos_editor_export_plugin.gd").new("khronos", plugin_version)
```
Comment (m4gr3d) [inline demo/addons/godotopenxr/.export/godot_openxr_editor_plugin.gd:20]: Seems like you inadvertently reverted the use of the `globals` constants.

**Review by m4gr3d — CHANGES_REQUESTED** [review]
Most of the changes look good! Just a minor comment to address, and the PR should be rebased.

**Comment by m4gr3d** [comment]
Superseded by https://github.com/GodotVR/godot_openxr_loaders/pull/55

**Pull request closed without merging**